infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/fsm/LogReplicationFSM.java

```diff
@@ -290,26 +289,26 @@ private void consume() {
             LogReplicationEvent event = eventQueue.take();
 
             if (event.getType() == LogReplicationEventType.REPLICATION_START) {
-                baseSnapshot = event.getMetadata().getSyncTimestamp();
-                ackedTimestamp = baseSnapshot;
+                baseSnapshot = event.getMetadata().getLastBaseSnapshot();
```

Comment:
I had questions about it in PR, looks like now its fixed, cool

Reply:
yes, most likely you were hitting this bug as well!